src/docs/index.less

```diff
@@ -77,3 +78,222 @@ html {
 		}
 	}
 }
+
```

Comment:
should these styles perhaps live in their own less file at `src/docs/containers/colors.less` maybe?